helm/superset/values.yaml

```diff
@@ -113,6 +113,20 @@ extraVolumeMounts: []
  # - name: additionalSecret:
  #   mountPath: /mnt/secret
 
+# livenessProbe:
```

Comment:
These settings should be nested under the `supersetNode` config element

Reply:
Thanks for you comment, i changed these ;)